qualtrics_link/templates/qualtrics_link/main.html

```diff
@@ -91,6 +97,35 @@ <h4>Enter an HUID to spoof</h4>
 	</tr>
 </table>
 
+<div class="well qualtrics_admin">
+	<h4>Manage Qualtrics Division and/or Role</h4>
+	{% if not qualtrics_user_in_db %}
+		<div class="alert alert-warning" role="alert">
+			This user is not currently in our Postgres table. Please contact a dev for further assistance.
+		</div>
+	{% endif %}
+	<form action="internal" method="POST">{% csrf_token %}
+		<div class="col-sm-4">
+			Division: {{ qualtrics_user_update_form.division }}
+		</div>
+		<div class="col-sm-3">
+			Role: {{ qualtrics_user_update_form.role }}
+		</div>
+		<div class="col-sm-4">
+			Skip in update job: {{ qualtrics_user_update_form.manually_updated }}
+		</div>
+		<input type="hidden" name="huid" value='{{ huid }}'>
+		<div class="col-sm-1">
+			{% if not qualtrics_user_in_db %}
+				<input type="submit" value="Submit" class="btn btn-primary disabled">
```

Comment:
When the user  is not found, it is greyed out but I am able to still submit. (It's not updating the DB but is reloading the screen)